### PR TITLE
fix: add ZahraShahid

### LINKED ISSUE #1450: ZahraShahid
## Profile Screenshot

zash_github

### PR DESCRIPTION
**Pull request by ZahraShahid**
resolves #1450 

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [x] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/ZahraShahid.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/65255043/193467864-9c1d9364-b086-4e64-95bc-c46d68a5ef48.png",
+  "issueId": 1450,
+  "name": "Zahra Shahid",
+  "username": "ZahraShahid"
+}
```
Comment (Panquesito7) [inline profiles/ZahraShahid.json:6]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

Comment (ZahraShahid) [reply profiles/ZahraShahid.json]: Resolved

File: profiles/ZahraShahid.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "image": "https://user-images.githubusercontent.com/65255043/193467864-9c1d9364-b086-4e64-95bc-c46d68a5ef48.png",
+  "issueId": 1450,
+  "name": "Zahra Shahid",
+  "username": "ZahraShahid"
+
```
Comment (Panquesito7) [inline profiles/ZahraShahid.json:6]: Extra unneeded line here.
```suggestion
```

Comment (ZahraShahid) [reply profiles/ZahraShahid.json]: resolved.

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thanks! 🎉

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**